global_costmap:
   robot_base_frame: base_footprint
   update_frequency: 5.0
   publish_frequency: 5.0
   static_map: true
   transform_tolerance: 0.25
   footprint_padding: 0.05
   static_layer: {map_topic: /map}